en/developer_guide/extend_document.rst

```diff
@@ -0,0 +1,60 @@
+Extend document and repository
+==============================
+
+Use case
+----------
+
+OpenOrchestra provides users and developpers with a set of entities, for example the content entity, functionnal kernel of a CMS.
+For specific use, It could be necessary to extend the model of an entity and its repository, i.e. the class which allows to load the instances of this entity.
+ 
+
+How to
+------
+
+These extensions will be done by creating extended classes for document (ODM context) or repository.
+To reference these classes, the following lines will be added to config.yml.
+
+.. code-block:: yaml
+
+    open_orchestra_model:
+        document:
+            content:
+                class: "MyBundle\Document\Content"
+                repository: "MyBundle\Repository\ContentRepository"
+ 
+The header of the two extended classes could be :
+ 
+Extended Content
+~~~~~~~~~~~~~~~~
+
+ .. code-block:: php
+
+    <?php
+    namespace MyBundle\Document;
+    
+    use OpenOrchestra\ModelBundle\Document\Content as BaseContent;
+    use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+    /**
+     * Extended Class Content
+     * @ODM\Document(
+     *   collection="content",
+     *   repositoryClass="MyBundle\Repository\ContentRepository"
+     * )
+     */
+    class Content extends BaseContent
+    {
+
+Extended Content Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
What to do if I only want to extend the ContentRepository class ?
I mean the annotations in the Content document class makes a reference on the Orchestra Content Repository Class :

    * @ODM\Document(
    *   collection="content",
    *   repositoryClass="OpenOrchestra\ModelBundle\Repository\ContentRepository"
    * )